core/trino-main/src/main/java/io/trino/server/ui/TrimmedBasicQueryInfo.java

```diff
@@ -103,6 +105,12 @@ public Optional<String> getSessionSource()
         return sessionSource;
     }
 
+    @JsonProperty
```

Comment:
Please create separate commits for each info that you're adding

Reply:
> Please create separate commits for each info that you're adding

Sure, I will separate commits after fixing the CICD errors~

Reply:
@raunaqmorarka Should we create each component change between backend/frontend change ?
For example like this:
Commit 1: Add client info in the query lists of Cluster Overview